testing/e2e/e2e_staking_test.go

```diff
@@ -115,12 +115,14 @@ func (s *BeaconKitE2ESuite) TestDepositRobustness() {
 
 	// Grab genesis validators to get withdrawal creds.
 	s.Require().NoError(apiClient.Connect(s.Ctx()))
-	s.Require().NotNil(apiClient.BeaconKitNodeClient)
-	response, err := apiClient.BeaconKitNodeClient.Validators(s.Ctx(), &api.ValidatorsOpts{
+	response, err := apiClient.Validators(s.Ctx(), &api.ValidatorsOpts{
 		State:   "genesis",
 		Indices: []phase0.ValidatorIndex{0, 1, 2, 3, 4},
 	})
 	s.Require().NoError(err)
+	s.Require().NotNil(response, "Validators response should not be nil")
+	s.Require().NotNil(response.Data, "Validators data should not be nil")
+
```

Comment:
this is the same check we are doing in line 127 right? One can be removed

Reply:
this agreed is a duplicate check. Would remove one of them

Reply:
done, removed one